software/controller/src/main.cpp

```diff
@@ -151,6 +153,9 @@ static void background_loop() {
 
     comms_handler(local_controller_status, &gui_status);
 
+    alarm.Handler(Hal.now(), CommsGetLastRxTime()) ? Hal.BuzzerOn()
```

Comment:
This is very nice! :)

Reply:
added comment next to variable, to all variables in class.